src/main/java/org/killbill/billing/catalog/api/Currency.java

```diff
@@ -21,173 +21,183 @@
 @XmlEnum
 public enum Currency {
     // Fiat currencies
-    AED,
-    AFN,
-    ALL,
-    AMD,
-    ANG,
-    AOA,
-    ARS,
-    AUD,
-    AWG,
-    AZN,
-    BAM,
-    BBD,
-    BDT,
-    BGN,
-    BHD,
-    BIF,
-    BMD,
-    BND,
-    BOB,
-    BRL,
-    BSD,
-    BTN,
-    BWP,
-    BYR,
-    BZD,
-    CAD,
-    CDF,
-    CHF,
-    CLP,
-    CNY,
-    COP,
-    CRC,
-    CUC,
-    CUP,
-    CVE,
-    CZK,
-    DJF,
-    DKK,
-    DOP,
-    DZD,
-    EGP,
-    ERN,
-    ETB,
-    EUR,
-    FJD,
-    FKP,
-    GBP,
-    GEL,
-    GGP,
-    GHS,
-    GIP,
-    GMD,
-    GNF,
-    GTQ,
-    GYD,
-    HKD,
-    HNL,
-    HRK,
-    HTG,
-    HUF,
-    IDR,
-    ILS,
-    IMP,
-    INR,
-    IQD,
-    IRR,
-    ISK,
-    JEP,
-    JMD,
-    JOD,
-    JPY,
-    KES,
-    KGS,
-    KHR,
-    KMF,
-    KPW,
-    KRW,
-    KWD,
-    KYD,
-    KZT,
-    LAK,
-    LBP,
-    LKR,
-    LRD,
-    LSL,
-    LTL,
-    LVL,
-    LYD,
-    MAD,
-    MDL,
-    MGA,
-    MKD,
-    MMK,
-    MNT,
-    MOP,
-    MRO,
-    MUR,
-    MVR,
-    MWK,
-    MXN,
-    MYR,
-    MZN,
-    NAD,
-    NGN,
-    NIO,
-    NOK,
-    NPR,
-    NZD,
-    OMR,
-    PAB,
-    PEN,
-    PGK,
-    PHP,
-    PKR,
-    PLN,
-    PYG,
-    QAR,
-    RON,
-    RSD,
-    RUB,
-    RWF,
-    SAR,
-    SBD,
-    SCR,
-    SDG,
-    SEK,
-    SGD,
-    SHP,
-    SLL,
-    SOS,
-    SPL,
-    SRD,
-    STD,
-    SVC,
-    SYP,
-    SZL,
-    THB,
-    TJS,
-    TMT,
-    TND,
-    TOP,
-    TRY,
-    TTD,
-    TVD,
-    TWD,
-    TZS,
-    UAH,
-    UGX,
-    USD,
-    UYU,
-    UZS,
-    VEF,
-    VND,
-    VUV,
-    WST,
-    XAF,
-    XCD,
-    XDR,
-    XOF,
-    XPF,
-    YER,
-    ZAR,
-    ZMW,
-    ZWD,
+    AED("د.إ"),
+    AFN("؋"),
+    ALL("L"),
+    AMD("դր."),
+    ANG("ƒ"),
+    AOA("Kz"),
+    ARS("$"),
+    AUD("A$"),
+    AWG("ƒ"),
+    AZN("ман"),
+    BAM("KM"),
+    BBD("Bds$"),
+    BDT("৳"),
+    BGN("лв"),
+    BHD(".د.ب"),
+    BIF("FBu"),
+    BMD("BD$"),
+    BND("B$"),
+    BOB("Bs."),
+    BRL("R$"),
+    BSD("B$"),
+    BTN("Nu."),
+    BWP("P"),
+    BYR("Br"),
+    BZD("BZ$"),
+    CAD("Can$"),
+    CDF("FC"),
+    CHF("SFr"),
+    CLP("CLP$"),
+    CNY("元/圆"),
+    COP("COL$"),
+    CRC("₡"),
+    CUC("CUC$"),
+    CUP("$MN"),
+    CVE("Esc"),
+    CZK("Kč"),
+    DJF("DF"),
+    DKK("Dkr"),
+    DOP("RD$"),
+    DZD("دج"),
+    EGP("ج.م."),
+    ERN("Nfk"),
+    ETB("Br"),
+    EUR("€"),
+    FJD("FJ$"),
+    FKP("FK£"),
+    GBP("£"),
+    GEL("ლ"),
+    GGP("£"),
+    GHS("GH₵"),
+    GIP("£"),
+    GMD("D"),
+    GNF("GFr"),
+    GTQ("Q"),
+    GYD("G$"),
+    HKD("元/圓"),
+    HNL("L"),
+    HRK("kn"),
+    HTG("G"),
+    HUF("Ft"),
+    IDR("Rp"),
+    ILS("₪"),
+    IMP("M£"),
+    INR("₹"),
+    IQD("د.ع"),
+    IRR("﷼"),
+    ISK("Íkr"),
+    JEP("£"),
+    JMD("J$"),
+    JOD("JD"),
+    JPY("¥"),
+    KES("Ksh"),
+    KGS("сом"),
+    KHR("៛"),
+    KMF("CF"),
+    KPW("₩"),
+    KRW("₩"),
+    KWD("د.ك"),
+    KYD("CI$"),
+    KZT("₸"),
```

Comment:
Interesting - I now see the proper symbol (₸) after you last push. Before, it was just a square.